tests/e2e/dsc_creation_test.go

```diff
@@ -424,6 +430,23 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testMRServiceMeshMember() error {
+	if tc.testDsc.Spec.Components.ModelRegistry.ManagementState != operatorv1.Managed {
+		return nil
+	}
+
+	// Get unstructured ServiceMeshMember
+	smm := unstructured.Unstructured{}
+	smm.SetAPIVersion("maistra.io/v1")
+	smm.SetKind("ServiceMeshMember")
+	err := tc.customClient.Get(tc.ctx,
+		client.ObjectKey{Namespace: modelregistry.ModelRegistriesNamespace, Name: "default"}, &smm)
+	if err != nil {
+		return fmt.Errorf("failed to get servicemesh member: %w", err)
+	}
+	return nil
+}
```

Comment:
I am not quite sure how it fits to `e2e` category, but this is not part of the code I am familiar with. 

It would be better to check somehow if Istio configured what you need instead of checking the existence of an arbitrary resource without ensuring that behaviour your need is provided.

Reply:
The e2e test here starts the dsc along with istio, etc. so it's the easiest place to test this. It used to be that there were component specific tests here, but that's been kinda merged all into more generic checks. Which is ok, but there is still a need to run component specific checks. 
I'm sure we can refactor this into something better eventually.